miniupnpd/netfilter_nft/nftnlrdr.c

```diff
@@ -177,13 +183,16 @@ add_filter_rule2(const char * ifname,
 
 	d_printf(("add_filter_rule2(%s, %s, %s, %d, %d, %d, %s)\n",
 	          ifname, rhost, iaddr, eport, iport, proto, desc));
+
 	if (rhost != NULL && strcmp(rhost, "") != 0) {
-            rhost_addr = inet_addr(rhost);
-        }
-	r = rule_set_filter(NFPROTO_IPV4, ifname, proto,
-			    rhost_addr, inet_addr(iaddr), eport, iport,
-			    desc, 0);
-	return nft_send_request(r, NFT_MSG_NEWRULE);
+        rhost_addr = inet_addr(rhost);
+    }
+
+	r = rule_set_filter(NFPROTO_INET, ifname, proto,
+			    rhost_addr, inet_addr(iaddr),
+				eport, iport, 0, desc, 0);
+
+	return nft_send_request(r, NFT_MSG_NEWRULE, RULE_CHAIN_FILTER);
```

Comment:
it is difficult to understand what is changed (many changes for just indentation, etc.)